automated install/basic-install.sh

```diff
@@ -175,59 +175,77 @@ is_command() {
-        display_warning=false
-    fi
+        if [ "$display_warning" = true ]; then
+            printf "  %b %bUnsupported OS detected%b\\n" "${CROSS}" "${COL_LIGHT_RED}" "${COL_NC}"
 
-    if [ "$display_warning" = true ] && [ "$PIHOLE_SKIP_OS_CHECK" != true ]; then
-        printf "  %b %bUnsupported OS detected%b\\n" "${CROSS}" "${COL_LIGHT_RED}" "${COL_NC}"
-        printf "      https://docs.pi-hole.net/main/prerequesites/#supported-operating-systems\\n"
-        printf "\\n"
-        printf "      This check can be skipped by setting the environment variable %bPIHOLE_SKIP_OS_CHECK%b to %btrue%b\\n" "${COL_LIGHT_RED}" "${COL_NC}" "${COL_LIGHT_RED}" "${COL_NC}"
-        printf "      e.g: 'sudo PIHOLE_SKIP_OS_CHECK=true curl -sSL https://install.pi-hole.net | bash'\\n"
-        printf "      or   'sudo PIHOLE_SKIP_OS_CHECK=true pihole -up'\\n"
-        printf "      By setting this variable to true you acknowledge there may be issues with Pi-hole during or after the install\\n"
-        printf "      If that is the case, you can feel free to ask the community on Discourse with the %bCommunity Help%b category:\\n" "${COL_LIGHT_RED}" "${COL_NC}"
-        printf "      https://discourse.pi-hole.net/c/bugs-problems-issues/community-help/\\n"
-        exit 1
-    elif [ "$display_warning" = true ] && [ "$PIHOLE_SKIP_OS_CHECK" = true ]; then
-        printf "  %b %bUnsupported OS detected%b. PIHOLE_SKIP_OS_CHECK env variable set to true - installer will continue\\n" "${INFO}" "${COL_LIGHT_RED}" "${COL_NC}"
+            # Display findings of above test back to the user
+            if [ "$valid_os" = true ]; then
+                printf "  %b Distro: %b%b%b\\n" "${TICK}" "${COL_GREEN}" "${detected_os}" "${COL_NC}"
+                if [ "$valid_version" = true ]; then
+                    printf "  %b Version: %b%b%b\\n" "${TICK}" "${COL_GREEN}" "${detected_version}" "${COL_NC}"
+                else
+                    printf "  %b Version: %b%b%b\\n" "${CROSS}" "${COL_RED}" "${detected_version}" "${COL_NC}"
+                    printf "  %b Error: %b%b is supported but version %b is currently unsupported%b\\n" "${CROSS}" "${COL_RED}" "${detected_os}" "${detected_version}" "${COL_NC}"
+                fi
+            else
+                printf "  %b Distro: %b%b%b\\n" "${CROSS}" "${COL_RED}" "${detected_os}" "${COL_NC}"
+                printf "  %b Error: %b%b is not a supported distro%b\\n" "${CROSS}" "${COL_RED}" "${detected_os}" "${COL_NC}"
+            fi
+            printf "\\n"
+            printf "      https://docs.pi-hole.net/main/prerequesites/#supported-operating-systems\\n\\n"
+            printf "      This check can be skipped by setting the environment variable %bPIHOLE_SKIP_OS_CHECK%b to %btrue%b\\n" "${COL_LIGHT_RED}" "${COL_NC}" "${COL_LIGHT_RED}" "${COL_NC}"
+            printf "      e.g: 'sudo PIHOLE_SKIP_OS_CHECK=true curl -sSL https://install.pi-hole.net | bash'\\n"
```

Comment:
```suggestion
            printf "      e.g: 'curl -sSL https://install.pi-hole.net | PIHOLE_SKIP_OS_CHECK=true sudo bash'\\n"
```
The ENVVAR needs to be set in the bash environment executing the script.

Possibly `export PIHOLE_SKIP_OS_CHECK=true; curl -sSL https://install.pi-hole.net | bash` but I'd have to play around with it more.